docker/base/Dockerfile.j2

```diff
@@ -416,16 +422,7 @@ COPY httpd_setup.sh /usr/local/bin/kolla_httpd_setup
 COPY sudoers /etc/sudoers
 
 {% if use_dumb_init %}
-
-{% block dumb_init_installation %}
```

Comment:
I guess we broke anyone who overrides this block...

Reply:
Could add it back as an empty block?

Reply:
We could do, although the installation method has changed so it may be best if users remove it.